gwswlib/threedi.py

```diff
@@ -236,10 +333,9 @@ def get_code(self, connection, default_code=""):
         ]
```

Comment:
Ik snap niet wat hier gebeurt. Je plakt twee codes eerst aan elkaar met een streepje en gaat daarna bij andere elementen kijken of "code" samengesteld is uit iets met minimaal twee streepjes en je pakt dan het eerste setje?

Klopt het dat je dit op basis van een samengesteld stringetje doet? I.p.v. op basis van de eigenlijke data waarop dat stringetje (hopelijk) gebaseerd is?

De `rpartition` (die moest ik googlen :-) ) kan makkelijker geschreven worden als `if element["code"].startswith(element_codes)`.

Op zich leest daar de `element_codes` weer lastig omdat het een meervoud is. `combined_code_string` ofzo?